runtime/near-vm-runner/src/tests/compile_errors.rs

```diff
@@ -149,7 +150,7 @@ fn slow_test_limit_contract_functions_number() {
         .make(),
     )
     .protocol_features(&[
-        ProtocolFeature::LimitContractFunctionsNumber,
```

Comment:
(aside) @nagisa, how do we handle tests here after protocol version deprecation? Is it fine to remove checks for these protocol versions?

Reply:
(aside) same answer here as in #13267.

Reply:
Revert this test